meldet-client/prisma/reports-seed-data.json

```diff
@@ -0,0 +1,16676 @@
+        
+        "isPrivate": false,
+        "hasSocialMediaConsent": true,
+        "categories": [
+            9,
+            22,
+            24,
+            28
+        ]
+    },
+    {
+        "createdAt": "2021-11-23T20:07:29.016Z",
+        "incidentDate": "28/2/2021",
+        "address": "",
+        "lat": "51.05594379999999",
+        "lng": "3.7285518",
+        "title": "Fuck Charlatan",
+        "description": "I was kissing a girl I was dating on the dancefloor and a man starten filming us ",
+        
+        "isPrivate": false,
+        "hasSocialMediaConsent": true,
+        "categories": [
+            15,
+            17,
+            18,
+            21
+        ]
+    },
+    {
+        "createdAt": "2021-11-23T20:07:29.016Z",
+        "incidentDate": "23/11/2021",
+        "address": "",
+        "lat": "51.04918439999999",
+        "lng": "3.7346602",
+        "title": "",
+        "description": "Reacted to someone who was catcalling me, they got mad and spit in my face",
+        
+        "isPrivate": false,
+        "hasSocialMediaConsent": true,
+        "categories": [
+            9,
+            18,
+            24
+        ]
+    },
+    {
+        "createdAt": "2021-11-23T20:07:29.016Z",
+        "incidentDate": "13/5/2021",
+        "address": "",
+        "lat": "51.0482017",
+        "lng": "3.7227318",
+        "title": "1312",
+        "description": "Was walking to class, a (drunk) man walked behind me. He tapped me on the shoulder and as I turned around he grabbed my breasts. I headbutted him and started shouting. A woman came out of her house to see what was going on. She called the cops. The three of us waited. The man with a smug face. When the cops arrived, they were friendly up until the point they realised the woman didn\\'t see anything happen, she only heard me scream. Then they changed the narrative and became condescending towards me. The man denied everything. They asked me \\\"What do you want us to do, madammeke?\\\" and even went as far to say that I was lucky the man didn\\'t report me because the only thing the could physically see was that I headbumped him. They made no record of whatever happened. We all just walked away. They pretty much just laughed in my face.",
+        
+        "isPrivate": false,
+        "hasSocialMediaConsent": true,
```

Comment:
This is just dummy data though, right? Otherwise, I wouldn't necessarily default the `hasSocialMediaConsent` flag to true for previous reports

Reply:
The data is used as dummy data, but it is based on a datadump from the real database 2 weeks ago. Your comment leaves me thinking: what will we do when we migrate the database? Maybe the field should change from a boolean to an enum: 
```
socialMediaConsent: 
ACCEPTED
DECLINED
UNKNOWN
```